util/program-names/program-names.json

```diff
@@ -14,6 +14,7 @@
   "G0059": "G0059",
   "M0001": "M0001",
   "M0002": "M0002",
+  "M0003": "M0003",
```

Comment:
@peg-uehling Added M0003 back to our program-names.json object - my understanding is that this JSON object plus the enum and interface defined in program-names.ts should include _all_ program names, past and present. In this case, M0003 was valid in 2024, but not in 2025.